src/ms-review.tex

```diff
@@ -346,18 +346,18 @@ \section{Introduction}
 %%% Question: starting date? Is is relevant, what is the real start?
 The \gammapy project was started following the model of \astropy, with the objective of building a common
 software library for very high-energy \gammaray data analysis \citep{gammapy_2015}. 
-The core of the idea is illustrated in Figure~\ref{fig:big_picture}. Various \gammaray instruments
-export their data to the standardised common \gadf data format. This data can then be combined and
+The core of the idea is illustrated in Figure~\ref{fig:big_picture}. The various \gammaray instruments
+can export their data to a common data format (\gadf) and then these data can be combined and
 analysed using a common software library.
-This means that the \gammapy package is not specific to any instrument,
-but an independent community-developed software project.
+The \gammapy is package an independent community-developed software project, it has been selected to be the core library for the Science Analysis tools of CTAO
+but also involves contributors associated to other instruments.
```

Comment:
- related comment:
clarification of "Gammapy is not part of any instrument" vs. "Gammapy has been chosen as base of the CTAO science analysis tool"